src/js/services/optimizely.js

```diff
@@ -267,4 +276,17 @@ export const storeExperimentParticipation = (
   }
 };
 
+export const trackExperimentEntry = (userId, orgId, projectId) => {
```

Comment:
Is there any reason why we are exporting this function? We dont seem to be importing it anywhere and you are initializing `getTrackExperimentEntry` in `app.js` which has `trackExperimentEntry` in its function body.

Reply:
code updated thanks!

Reply:
I exported it to be reusable but code has been updated. You are correct it wasn't needed.